x-pack/plugins/observability/public/components/alerts_table/slo/get_slo_alerts_table_configuration.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { GetRenderCellValue } from '@kbn/triggers-actions-ui-plugin/public';
+import { TIMESTAMP } from '@kbn/rule-data-utils';
+import { SortOrder } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import { AlertsTableConfigurationRegistry } from '@kbn/triggers-actions-ui-plugin/public/types';
+import { AlertConsumers } from '@kbn/rule-data-utils';
+import { casesFeatureId, observabilityFeatureId } from '../../../../common';
+import { getRenderCellValue } from './render_cell_value';
+import { columns } from './default_columns';
+import { useGetAlertFlyoutComponents } from '../../alerts_flyout/use_get_alert_flyout_components';
+import type { ObservabilityRuleTypeRegistry } from '../../../rules/create_observability_rule_type_registry';
+import type { ConfigSchema } from '../../../plugin';
+import type { TopAlert } from '../../../typings/alerts';
+
+export const getSloAlertsTableConfiguration = (
```

Comment:
I am using a custom alerts table configuration for the SLO embeddable alerts table. I noticed a weird behavior with the table loading indicator not going away when I switched to the custom alerts table configuration. When using the default o11y alerts table configuration, I didn't notice any issue with the loading indicator. Custom configuration and loading indicator issue, might not be related though. This is just an observation. I need to check further what is going on with this.

Reply:
https://github.com/elastic/kibana/assets/2852703/5b0f1038-d4ca-428e-8b2f-83b4ae4a3be5

Reply:
Follow up ticket to check loading indicator issue https://github.com/elastic/kibana/issues/171738